modules/virt-about-hco-operator.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-architecture.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-hco-operator_{context}"]
+= About hco-operator
+
+The `hco-operator` (HCO) provides a single entry  point for deploying and managing KubeVirt and several helper operators with opinionated defaults, and creates custom resources (CRs) for those operators.
+
+image::cnv_components_hco-operator.png[hco-operator components]
+
+.hco-operator components
+[cols="1,1"]
+|===
+|*Component* |*Description*
+
+|`deployment/hco-webhook`
+|Validates the `HyperConverged` custom resource contents.
+
+|`deployment/hyperconverged-cluster-cli-download`
+|Provides the `virtctl` tool binaries to the cluster so that you can download them directly from the cluster.
+
+|`KubeVirt/kubevirt-kubevirt-hyperconverged`
+|Contains all operators, CRs, and objects needed by KubeVirt.
```

Comment:
I think s/KubeVirt/{VirtProducName} in the description. You can check with the SME to be sure.

Reply:
@tiraboschi Please see the note here. Should this definition use KubeVirt or OpenShift Virtualization?

Reply:
`OpenShift Virtualization`